lib/pbench/server/auth/__init__.py

```diff
@@ -468,7 +468,7 @@ def token_introspect_offline(self, token: str) -> JSON:
                 token,
                 self._pem_public_key,
                 algorithms=[self._TOKEN_ALG],
-                audience=self.client_id,
+                audience=["account", self.client_id],
```

Comment:
Where is this audience coming from?

Reply:
Oops, I forgot to add a comment here. 
Basically, if a user logs in with a username and password directly with a Keycloak rest API using private client then the `aud` claim included in the token is `account` 
for example this request:
```
requests.post(
        "<token_endpoint>",
        headers=<{headers}>,
        data={"client_id": "pbench-server-client", "client_secret": <secret>,
              "scope": "profile email", "grant_type": "password", "username": user, "password": "123"},
        verify=False
    )
```
will generate a token that has a payload like following:
```
{'exp': 1675890382, 
'iat': 1675890082,
'jti': 'a8dc9cca-d734-4758-ab1d-be6510fe85b2', 
'iss': 'http://localhost:8090/realms/pbench-server', 
'aud': 'account', 
'sub': '9202c4a9-5901-46dc-9b8e-2643a5f4372e', 
'typ': 'Bearer', 
'azp': 'pbench-server-client', 
'session_state': '5730b4b1-bdb3-438c-a308-d3c1f8108fd8', ....}

```
And right now our functional test user is logging in using the Keycloak Rest API, so I had to include this claim to get the functional tests to pass.

Reply:
This one should be fixed now. I have added a client scope in the Keycloak configuration that will add the Pbench openid client (pbench-dashboard) in the `aud` claim by default. So the functional test user tokens should be able to get validated now.

Reply:
@portante can we resolve this comment?

Reply:
We don't require comments to be resolved before merging.  Forgive me, but I have not had the time to review the changes in detail.

Reply:
In that case, can you consider removing your objection to this PR, its blocking the merge.